addons/binding/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/handler/MiIoAbstractHandler.java

```diff
@@ -342,4 +357,68 @@ protected JsonObject getJsonResultHelper(String response) {
         }
         return null;
     }
+
+    private boolean buildChannelStructure(String deviceName) {
```

Comment:
I didn't think you are planning on implementing special handlers at all.
Even then it's not bad to have it in place with the thing definition.
But like I said, it's your call 👍